src/content/guides/progressive-web-application.md

```diff
@@ -135,7 +135,7 @@ __index.js__
 
 + if ('serviceWorker' in navigator) {
 +   window.addEventListener('load', () => {
-+     navigator.serviceWorker.register('/sw.js').then(registration => {
++     navigator.serviceWorker.register('/service-worker.js').then(registration => {
```

Comment:
did you run the guide from scratch?

As i see there is an output on the page which says that `sw.js` with a size of 1kb was emited.